server/middleware/get-contract-by-id-from-param.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const getContractByID = require('../lib/get-contract-by-id');
+
+
+module.exports = (req, res, next) => {
+	const {contract_id} = req.params || {};
+	if (!contract_id) {
+		return res.sendStatus(400);
```

Comment:
Could this maybe send a message as well like, `contract_id not provided` or something like that?

Reply:
done and dusted